sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/DecryptorV2.java

```diff
@@ -45,6 +45,7 @@ protected DecryptorV2(AsyncKeyEncryptionKeyResolver keyResolver, AsyncKeyEncrypt
     Flux<ByteBuffer> decrypt(Flux<ByteBuffer> encryptedFlux, EncryptedBlobRange encryptedBlobRange,
         boolean padding, String requestUri, AtomicLong totalInputBytes, byte[] contentEncryptionKey) {
         // Buffer an exact region with the nonce and tag
+        // this needs to be a configurable value
```

Comment:
We should probably remove this added comment as of now we are not planning to touch the decryption logic. This class will always be pulling the region length from the encryption metadata on the blob being downloaded and we do not want to be using the configuration from the client.